flatgeobuf/datatypes.md

```diff
@@ -0,0 +1,30 @@
+# Mapping of fiboa Data Types to FlatGeobuf
+
+The following table shows the data types that are used by fiboa in the Property definitions.
+It also shows the mapping to the [FlatGeobuf data types](https://github.com/flatgeobuf/flatgeobuf/blob/ee7c8f5f45c67dd4a84a51fef518dfebc3e19d0a/src/fbs/header.fbs#L26-L42).
+
+| fiboa Schema data type                              | FlatGeobuf                                                 |
+| --------------------------------------------------- | ------------------------------------------------------------ |
+| boolean                                             | Bool                                                       |
+| int8                                                | Byte                                                       |
+| uint8                                               | UByte |
+| int16                                               | Short |
+| uint16                                              | UShort |
+| int32                                               | Int |
+| uint32                                              | UInt |
+| int64                                               | Long |
+| uint64                                              | ULong |
+| float<br />IEEE 32-bit                              | Float                                                        |
+| double<br />IEEE 64-bit                             | Double                                                       |
+| binary                                              | Binary                                                   |
+| string<br />charset: UTF-8                          | String                                          |
+| array                                               | Json Array                                                         |
+| object<br />keys: string<br />values: any           | Json Object                                                 |
+| date                                                | DateTime (ISO 8601 date)                                                 |
+| date-time<br />with milliseconds<br />timezone: UTC | DateTime (ISO 8601 date time) |
```

Comment:
I couldn't find datetime data types in the specification. Can you help me finding it? Otherwise it should probably string with a pre-defined format (e.g. ISO 8601 / RFC 3339)

Reply:
Yes, I was refering to this line: https://github.com/flatgeobuf/flatgeobuf/blob/2828e07fd759aa8b44c6b1aaab41320b22dd0d29/src/fbs/header.fbs#L40 It is (probably) an ISO 8601 string, but it is it's own `ColumnType` in the FlatGeobuf schema.

Related comment: https://github.com/fiboa/specification/pull/32#discussion_r1634510400

Reply:
Yes, I agree `DateTime` is not well defined. I put a link to the allowed column types here: https://github.com/fiboa/specification/pull/32/files#diff-c33cc7b35504997d9be5d9db7ecaf8f5975eef4b5fe303b28c915033bfba02adR4 But we can also link to https://unpkg.com/flatgeobuf@3.31.1/dist/doc/enums/generic.ColumnType.html if you prefer.

Reply:
Didn't see that link before, looks good. So here the Date issue is still open.